### ISSUE TITLE
PyIDM failed to obtain the real download URL for orca. 

### ISSUE THREAD

**Issue by hongyi-zhao** [pyIDM/PyIDM#141]
Hi,

I try to download [orca](https://orcaforum.kofo.mpg.de/app.php/dlext/?cat=10) with PyIDM. Firstly, I downloaded the [ORCA 4.2.1, Linux, x86-64, .tar.xz](https://orcaforum.kofo.mpg.de/app.php/dlext/?view=detail&df_id=69&sid=5a1e853c6566a545859164d47f0d78b6) with firefox and it will show the following downloading URL: https://orcaforum.kofo.mpg.de/app.php/dlext/?sid=f4f1a5043c888e529297974d69b21d8a.

But when I feed this URL to PyIDM, and PyIDM failed to obtain the real download URL for orca. 

Any hints for this problem?

Regards,
HY

**Comment by pyIDM** [comment]
this is what i'm getting when opening the link you provided
there is no file to download

![image](https://user-images.githubusercontent.com/58998813/88310730-4d672e80-cd10-11ea-8c13-fe49346714c6.png)

if you need to login you should pass login credentials to PyIDM in Settings

**Comment by hongyi-zhao** [comment]
Yes, it needs registration first. But  now I've filled out the  login credentials in PyIDM's `Website Auth`  Settings, still PyIDM failed to grab the real URL for that file. See the following appeared on the official website when login correcly:


![image](https://user-images.githubusercontent.com/11155854/88352441-55fd4a80-cd8c-11ea-9872-3ca8b361c53a.png)
